don't hardcode path to include/gatb/gatb_core.hpp so we can use GATB-Core provided as dependency
author: Kenneth Hoste (HPC-UGent)
--- lordec-src_0.9/Makefile.orig	2023-03-31 09:54:40.838204432 +0200
+++ lordec-src_0.9/Makefile	2023-03-31 09:55:42.987061551 +0200
@@ -63,7 +63,7 @@
 $(PROG_GRAPH) $(OBJS_TEST_CORRECT) $(OBJS_CATCHTEST_CORRECT) $(OBJS_CORRECT) $(OBJS_STATS) $(OBJS_TRIM) $(OBJS_TRIM_SPLIT): boost_include/boost/graph/graph_traits.hpp
 endif
 # ANYWAY we need GATB (compiled and installed) to compile all the objects
-$(PROG_GRAPH) $(OBJS_TEST_CORRECT) $(OBJS_CATCHTEST_CORRECT) $(OBJS_CORRECT) $(OBJS_STATS) $(OBJS_TRIM) $(OBJS_TRIM_SPLIT): gatb_v$(GATBVERSION)$(SUFFIX)/include/gatb/gatb_core.hpp
+$(PROG_GRAPH) $(OBJS_TEST_CORRECT) $(OBJS_CATCHTEST_CORRECT) $(OBJS_CORRECT) $(OBJS_STATS) $(OBJS_TRIM) $(OBJS_TRIM_SPLIT): $(GATB)/include/gatb/gatb_core.hpp
 
 # get version from a file
 VERSION := $(shell cat version.txt)
